samples/spm/README.rst

```diff
@@ -51,28 +56,13 @@ Requirements for the application firmware
 
 * The application firmware must be built as a non-secure firmware for the build target (for example, ``nrf9160dk_nrf9160_ns`` for the nRF9160 DK).
 
-Automatic building of SPM when SPM is enabled
-=============================================
-
-The sample is automatically built by the non-secure applications when the non-secure build target is used (for example, ``nrf9160dk_nrf9160_ns``).
-However, it is not a part of the non-secure application.
-
-Instead of programming SPM and the non-secure application at the same time, you might want to program them individually.
-To do this, disable the automatic building of SPM by setting the option ``CONFIG_SPM=n`` in the :file:`prj.conf` file of the application.
-
-If this results in a single-image build, the start address of the non-secure application will change.
-The security attribution configuration for the flash will change when SPM is not built as a sub-image.
-
 Building and running
 ********************
 
 .. |sample path| replace:: :file:`samples/spm`
 
 .. include:: /includes/build_and_run.txt
 
-The sample is built as a secure firmware image for the ``nrf9160dk_nrf9160`` and ``nrf5340dk_nrf5340`` build targets.
-See `Automatic building of SPM when SPM is enabled`_ if you want to program it independently from the non-secure application firmware.
-
```

Comment:
@hakonfam , @b-gent : As now TFM is the included as default for all applications and samples instead of SPM, shouldn't these sentences be removed?

Reply:
These should be removed, correct.

Reply:
@hakonfam - I have incorporated your comments. Could you review the PR again and approve the same if all looks good from your end.